WebP.xcodeproj/project.pbxproj

```diff
@@ -811,6 +811,7 @@
 			);
 			name = "Package Modules";
 			outputPaths = (
+				"$(DERIVED_FILE_DIR)/libWebP.a",
```

Comment:
If we specify the static lib as the output of the script, it can be smarter about when to run the script (and this silences the "script will run all the time" warning in a productive way other than telling it to shut up).